src/js/controllers/modals/search.js

```diff
@@ -50,8 +52,19 @@ angular.module('copayApp.controllers').controller('searchController', function($
         return lodash.includes(tx.searcheableString, search.toLowerCase());
       });
 
+      if (search) {
+        if ((search.indexOf('bitcoincash:') >= 0 || search[0] === 'C' || search[0] === 'H' || search[0] === 'p' || search[0] === 'q') && search.replace('bitcoincash:', '').length === 42) { // CashAddr
```

Comment:
If you're check exact length, then 'bitcoincash:' has to be at the start, not > 0, does it not?

Reply:
I agree, 'bitcoincash:' has to be at the start of the string, and the index check should be == 0.

Reply:
You are right, I fixed it.